server_installation/topics/profiles.adoc

```diff
@@ -34,6 +34,7 @@ The features that can be enabled and disabled are:
 * Authorization - authorization services
 * Impersonation - ability for admins to impersonate users
 * Script - write custom authenticators using JavaScript
+* Docker - use docker-v2 authentication protocol
```

Comment:
Should we mention that the docker profile is disabled by default in the community (unlike the all other profiles which are enabled)?